classes.js

```diff
@@ -0,0 +1,30 @@
+//class that is modeling data
```

Comment:
Is there a reason that this file is in your solo build?

Reply:
I'm not going to lie, I only barely remember writing that at all? Anyways, deleted! Thanks for catching that!